### PR TITLE
plugin.video.presstv 1.2.0

### PR DESCRIPTION
**Pull request by scriptik**
### Description
PressTV Watch Live  www.presstv.com

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.video.presstv/default.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+__settings__ = xbmcaddon.Addon()
+home = __settings__.getAddonInfo('path')
+addon_handle = int(sys.argv[1])
+icon = xbmc.translatePath(os.path.join(home, 'icon.png'))
+li = xbmcgui.ListItem ('Live Stream')
+li.setArt({ 'thumb': icon })
+li.setInfo('video', {'plot':'PressTV Live Stream'})
+
+liveVideoUrl = "https://live2.presstv.com/liveprs/smil:liveprs.smil/playlist.m3u8"
```
Comment (Lunatixz) [inline plugin.video.presstv/default.py]: Can you include parser code that scrapes this link from its origin website? THX

Comment (basrieter) [reply plugin.video.presstv/default.py]: @scriptik Where is this video stream hosted? In other words: where did you get the stream from?

Comment (Lunatixz) [reply plugin.video.presstv/default.py]: Link verified, okay as standalone.

File: plugin.video.presstv/addon.xml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+       provider-name="pezhman">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0"/>
+  </requires>
+  <extension point="xbmc.python.pluginsource"
+            library="default.py">
+        <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en">PRESSTV</summary>
```
Comment (enen92) [inline plugin.video.presstv/addon.xml]: en_GB

File: plugin.video.presstv/addon.xml [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    <description lang="en">PRESS TV Addon for Kodi
```
Comment (enen92) [inline plugin.video.presstv/addon.xml]: en_GB

File: plugin.video.presstv/addon.xml [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+Press TV takes revolutionary steps as the first Iranian international news network,broadcasting in English on a round-the-clock basis.</description>
+    <platform>all</platform>
+    <language>fa</language>
+    <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
```
Comment (enen92) [inline plugin.video.presstv/addon.xml]: Possible to use a spdx identifier here: should be `GPL-2.0-only`

File: plugin.video.presstv/default.py [hunk 5]
```diff
@@ -0,0 +1,14 @@
+import xbmc, xbmcgui, os, xbmcaddon, xbmcplugin
+
+__settings__ = xbmcaddon.Addon()
+home = __settings__.getAddonInfo('path')
+addon_handle = int(sys.argv[1])
+icon = xbmc.translatePath(os.path.join(home, 'icon.png'))
+li = xbmcgui.ListItem ('Live Stream')
```
Comment (enen92) [inline plugin.video.presstv/default.py]: Please localize as this is presented to the user

File: plugin.video.presstv/default.py [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+li.setArt({ 'thumb': icon })
+li.setInfo('video', {'plot':'PressTV Live Stream'})
```
Comment (enen92) [inline plugin.video.presstv/default.py]: Same

File: plugin.video.presstv/addon.xml [hunk 7]
```diff
@@ -11,12 +11,12 @@
...
   <extension point="xbmc.addon.metadata">
-    <summary lang="en">PRESSTV</summary>
-    <description lang="en">PRESS TV Addon for Kodi
+    <summary lang="en_GB">PRESSTV</summary>
+    <description lang="en_GB">PRESS TV Addon for Kodi
 Press TV takes revolutionary steps as the first Iranian international news network,broadcasting in English on a round-the-clock basis.</description>
     <platform>all</platform>
     <language>fa</language>
-    <license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
+    <license>GNU General Public License v2.0 only</license>
```
Comment (enen92) [inline plugin.video.presstv/addon.xml]: `GPL-2.0-only`

File: plugin.video.presstv/addon.xml [hunk 8]
```diff
@@ -11,12 +11,12 @@
...
     <forum></forum>
```
Comment (enen92) [inline plugin.video.presstv/addon.xml]: drop if empty

Comment (scriptik) [reply plugin.video.presstv/addon.xml]: sorry , thanks

File: plugin.video.presstv/addon.xml [hunk 9]
```diff
@@ -15,7 +15,6 @@
     <description lang="en_GB">PRESS TV Addon for Kodi
 Press TV takes revolutionary steps as the first Iranian international news network,broadcasting in English on a round-the-clock basis.</description>
     <platform>all</platform>
-    <language>fa</language>
```
Comment (enen92) [inline plugin.video.presstv/addon.xml]: why drop the language tag? Shouldn't it be `en` ?

Comment (scriptik) [reply plugin.video.presstv/addon.xml]: Language tag added , thanks for your guidance

**Comment by enen92** [comment]
Ping @sc

**Pull request closed without merging**